src/plugins/abrt-journal.c

```diff
@@ -499,7 +499,7 @@ void abrt_journal_watch_notify_strings(abrt_journal_watch_t *watch, void *data)
     char message[JOURNALD_MAX_FIELD_SIZE + 1];
 
     if (abrt_journal_get_string_field(abrt_journal_watch_get_journal(watch), "MESSAGE", (char *)message) == NULL)
-        error_msg_and_die("Cannot read journal data.");
+        error_msg("Cannot read journal data, skipping.");
```

Comment:
Hm, I don't understand how this code works, but I have a stupid question: Doesn't this crash when `message` is read further down the code? I presume `message` is left unitialised when `abrt_journal_get_string_field()` fails so it might lead to an invalid read.